integration_tests/e2e/overview/postInductionOverview.cy.ts

```diff
@@ -0,0 +1,137 @@
+import Page from '../../pages/page'
+import CreateGoalPage from '../../pages/goal/CreateGoalPage'
+import OverviewPage from '../../pages/overview/OverviewPage'
+
+context('Prisoner Overview page - Post Induction', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignInAsUserWithEditAuthority')
+    cy.task('stubAuthUser')
+    cy.task('stubGetHeaderComponent')
+    cy.task('stubGetFooterComponent')
+    cy.task('stubPrisonerList')
+    cy.task('stubCiagInductionList')
+    cy.task('stubActionPlansList')
+    cy.task('getPrisonerById')
+    cy.task('stubGetShortQuestionSetCiagProfile')
+    cy.task('getActionPlan')
+    cy.task('stubLearnerProfile')
+    cy.task('stubLearnerEducation')
+  })
+
+  it('should render prisoner Overview page with Add Goal button given user has edit authority', () => {
```

Comment:
These tests are the ones moved from `overview.cy.ts`. The only difference is the addition of the CIAG Induction stub on line 16, so that these scenarios operate as post-induction